### PR TITLE
Satellite frontend tabs fixed

### PR DESCRIPTION
**Pull request by crawter**
What: 
some .vue files has corrupted whitespaces
Why:

Please describe the tests:
 - Test 1:
 - Test 2:
 
Please describe the performance impact:

## Code Review Checklist (to be filled out by reviewer)
 - [ ] Does the PR describe what changes are being made?
 - [ ] Does the PR describe why the changes are being made?
 - [ ] Does the code follow [our style guide](https://github.com/storj/docs/blob/master/code/Style.md)?
 - [ ] Does the code follow [our testing guide](https://github.com/storj/docs/blob/master/code/Testing.md)?
 - [ ] Is the PR appropriately sized? (If it could be broken into smaller PRs it should be)
 - [ ] Does the new code have enough tests? (*every* PR should have tests or justification otherwise. Bug-fix PRs especially)
 - [ ] Does the new code have enough documentation that answers "how do I use it?" and "what does it do?"? (both source documentation and [higher level](https://github.com/storj/docs), diagrams?)
 - [ ] Does any documentation need updating?
 - [ ] Do the database access patterns make sense?

### CODE REVIEW

File: web/satellite/src/components/account/ChangePasswordPopup.vue [hunk 1]
```diff
@@ -66,123 +66,118 @@
...
-                HeaderlessInput,
-                Button,
-            }
-        }
-    )
-
-    export default class ChangePasswordPopup extends Vue {
-    }
+    @Component({
+		data: function () {
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/account/ChangePasswordPopup.vue]: looks like not 4 spaces

File: web/satellite/src/components/account/billing/PaginationArea.vue [hunk 2]
```diff
@@ -1,17 +1,17 @@
-// Copyright (C) 2018 Storj Labs, Inc.
+// Copyright (C) 2019 Storj Labs, Inc.
 // See LICENSE for copying information.
 
 <template>
-	<div class="pagination-container">
-		<div class="pagination-container__pages">
+    <div class="pagination-container">
+        <div class="pagination-container__pages">
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/account/billing/PaginationArea.vue]: div looks like not aligned

File: web/satellite/src/components/buckets/BucketArea.vue [hunk 3]
```diff
@@ -2,27 +2,27 @@
...
-			<div class="buckets-header">
-				<p>Buckets</p>
-				<SearchArea/>
-			</div>
-			<div v-if="buckets.length > 0" class="buckets-container">
-				<table style="width:98.5%; margin-top:20px;">
-					<SortingHeader />
+    <div>
+        <div class="buckets-overflow" v-if="pages !== 0">
+    		<div class="buckets-header">
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/buckets/BucketArea.vue]: not aligned

File: web/satellite/src/components/dashboard/DashboardArea.vue [hunk 4]
```diff
@@ -14,23 +14,20 @@
...
-    )
+    })
 
-    export default class DashboardArea extends Vue {
-    }
+    export default class DashboardArea extends Vue {}
 </script>
 
-<style scoped lang="scss">
+<!--<style scoped lang="scss">-->
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/dashboard/DashboardArea.vue]: delete comments

File: web/satellite/src/components/project/UsageReport.vue [hunk 5]
```diff
@@ -57,146 +57,128 @@
...
-import ROUTES from '@/utils/constants/routerConstants';
-import Datepicker from '@/components/project/DatePicker.vue';
-import { NOTIFICATION_ACTIONS, PROJECT_USAGE_ACTIONS } from '@/utils/constants/actionNames';
-import { toUnixTimestamp } from '@/utils/time';
-
-	@Component(
-		{
-			mounted: function() {
-				this.$store.dispatch(PROJECT_USAGE_ACTIONS.FETCH_CURRENT_ROLLUP);
+	import { Component, Vue } from 'vue-property-decorator';
```
Comment (NikolaiYurchenko) [inline web/satellite/src/components/project/UsageReport.vue]: looks weird in github

File: web/satellite/src/components/common/RegistrationSuccessPopup.vue [hunk 6]
```diff
@@ -4,75 +4,75 @@
...
+    import { Component, Vue } from 'vue-property-decorator';
+    import Button from '@/components/common/Button.vue';
+    import { APP_STATE_ACTIONS } from '@/utils/constants/actionNames';
+    import ROUTES from '@/utils/constants/routerConstants';
+    import { resendEmailRequest } from '../../api/users';
+    import { getUserID } from '@/utils/consoleLocalStorage';
+
+
+    @Component(
+        {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/common/RegistrationSuccessPopup.vue]: "{" shouldn't start new line here

File: web/satellite/src/components/common/RegistrationSuccessPopup.vue [hunk 7]
```diff
@@ -4,75 +4,75 @@
...
-                }.bind(this), 1000);
-            }
-        },
-        components: {
-            Button,
-        },
-    }
-)
+        }
+    )
```
Comment (VitaliiShpital) [inline web/satellite/src/components/common/RegistrationSuccessPopup.vue]: ")" shouldn't start new line here

File: web/satellite/src/components/project/ProjectDetails.vue [hunk 8]
```diff
@@ -56,72 +56,69 @@
...
-                            description: this.$data.newDescription,
-                        }
-                    );
+            onSaveButtonClick: async function (): Promise<any> {
+                let response = await this.$store.dispatch(
+                    PROJETS_ACTIONS.UPDATE, {
+                        id: this.$store.getters.selectedProject.id,
+                        description: this.$data.newDescription
+                    }
+    		    );
```
Comment (VitaliiShpital) [inline web/satellite/src/components/project/ProjectDetails.vue]: ")" looks not aligned here

File: web/satellite/src/components/referral/ReferralStats.vue [hunk 9]
```diff
@@ -23,64 +23,64 @@
...
+              stats: {
+                 referred: {
+                     title: "referrals made",
+                     description: "People you referred who signed up",
+                     symbol: "",
+                     background: {
+                         backgroundColor: "#FFFFFF",
+                     }
+                 },
+                  availableCredits: {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/referral/ReferralStats.vue]: statement looks not aligned here

File: web/satellite/src/components/referral/ReferralStats.vue [hunk 10]
```diff
@@ -23,64 +23,64 @@
...
+                     title: "earned credits",
+                     description: "Free credits that will apply to your upcoming bill",
+                     symbol: "$",
+                   background: {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/referral/ReferralStats.vue]: statement looks not aligned here

File: web/satellite/src/components/referral/ReferralStats.vue [hunk 11]
```diff
@@ -23,64 +23,64 @@
...
+                         backgroundColor: "rgba(217, 225, 236, 0.5)",
+                     }
+                  },
+                  usedCredits: {
+                     title: "applied credits",
+                     description: "Free credits that have already been applied to your bill",
+                     symbol: "$",
+                   background: {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/referral/ReferralStats.vue]: statement looks not aligned here

File: web/satellite/src/components/team/AddUserPopup.vue [hunk 12]
```diff
@@ -63,162 +63,162 @@
...
+    import { Component, Vue } from 'vue-property-decorator';
+    import Button from '@/components/common/Button.vue';
+    import { EMPTY_STATE_IMAGES } from '@/utils/constants/emptyStatesImages';
+    import { PM_ACTIONS, NOTIFICATION_ACTIONS, APP_STATE_ACTIONS } from '@/utils/constants/actionNames';
+    import { EmailInput } from '@/types/EmailInput';
+    import { validateEmail } from '@/utils/validation';
+    import ROUTES from '@/utils/constants/routerConstants';
+
+    @Component(
+        {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/team/AddUserPopup.vue]: "{" shouldn't start new line here

File: web/satellite/src/components/team/AddUserPopup.vue [hunk 13]
```diff
@@ -63,162 +63,162 @@
...
+                },
+                onClose: function(): void {
+                    this.$store.dispatch(APP_STATE_ACTIONS.TOGGLE_TEAM_MEMBERS);
+                },
+                hasInputError: function (): boolean {
+                    return this.$data.inputs.some((element: EmailInput) => {
+                        return element.error;
+                    });
+                },
+             },
```
Comment (VitaliiShpital) [inline web/satellite/src/components/team/AddUserPopup.vue]: "}" looks not aligned here

File: web/satellite/src/components/team/AddUserPopup.vue [hunk 14]
```diff
@@ -63,162 +63,162 @@
...
+            components: {
+                Button
             }
-        },
-        components: {
-            Button
         }
-    }
-)
+    )
```
Comment (VitaliiShpital) [inline web/satellite/src/components/team/AddUserPopup.vue]: ")" should not start new line here

File: web/satellite/src/components/team/headerArea/SortDropdown.vue [hunk 15]
```diff
@@ -20,39 +20,39 @@
...
-        props: {
-            onClose: {
-                type: Function
-            }
-        },
-        methods: {
-            onCloseClick: function (): void {
-                this.$store.dispatch(APP_STATE_ACTIONS.TOGGLE_SORT_PM_BY_DROPDOWN);
+    @Component(
+        {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/team/headerArea/SortDropdown.vue]: "{" shouldn't start new line here

File: web/satellite/src/components/team/headerArea/SortDropdown.vue [hunk 16]
```diff
@@ -20,39 +20,39 @@
...
+                    this.$store.dispatch(NOTIFICATION_ACTIONS.ERROR, 'Unable to fetch project members');
 
-            }
-        },
-    }
-)
+                }
+            },
+        }
+    )
```
Comment (VitaliiShpital) [inline web/satellite/src/components/team/headerArea/SortDropdown.vue]: ")" shouldn't start new line here

File: web/satellite/src/components/common/Button.vue [hunk 17]
```diff
@@ -44,7 +43,7 @@ import { Component, Vue } from 'vue-property-decorator';
             },
             onPress: {
                 type: Function,
-                default: () => {
+                    default: () => {
```
Comment (VitaliiShpital) [inline web/satellite/src/components/common/Button.vue]: this tab shouldn't appear

**Pull request merged**